content/en/docs/download.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Download
+description: Officially supported Falco artifacts
+weight: 2
+---
+
+## Downloading
+
+The Falco Project community only supports two ways for downloading and running Falco:
+
+ - Running Falco directly on a Linux host
+ - Running the Falco userspace program in a container, with a driver installed on the underlying host.
+ 
+Below you can find artifacts for both. 
+
+
+### Download for Linux
+
+|        | development                                                                                                                 | stable                                                                                                              |
+|--------|-----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| rpm    | [![rpm-dev](https://img.shields.io/bintray/v/falcosecurity/rpm-dev/falco?label=Falco&color=%2300aec7&style=flat-square)][1] | [![rpm](https://img.shields.io/bintray/v/falcosecurity/rpm/falco?label=Falco&color=%23005763&style=flat-square)][2] |
+| deb    | [![deb-dev](https://img.shields.io/bintray/v/falcosecurity/deb-dev/falco?label=Falco&color=%2300aec7&style=flat-square)][3] | [![deb](https://img.shields.io/bintray/v/falcosecurity/deb/falco?label=Falco&color=%23005763&style=flat-square)][4] |
+| binary | [![bin-dev](https://img.shields.io/bintray/v/falcosecurity/bin-dev/falco?label=Falco&color=%2300aec7&style=flat-square)][5] | [![bin](https://img.shields.io/bintray/v/falcosecurity/bin/falco?label=Falco&color=%23005763&style=flat-square)][6] |
+
+The list of all available artifacts can be found [here](https://bintray.com/falcosecurity).
+
+---
+
+### Download container images
+
+{{< info >}}
+
+Falco depends on having a driver installed on the host system to get information about the running system calls.
+
+The preferred installation method is to install the driver using the native artifacts defined above or 
+temporarily run the `falcosecurity/falco-driver-loader` image as privileged, then using the `falcosecurity/falco-no-driver`.
+
+{{< /info >}}
+
+|tag | pull command | description |
```

Comment:
Can we call out which containers require privileges to escalate to the host? I hate to nit-pick this but it really is a big deal for a lot of production users. Maybe we just add a column at the end `requires-privileged` or something?

Reply:
After having talked with @leodido and @fntlnz last week, I believe that just a simple "privileged YES/NO" is not enough (for example, see [here](https://github.com/falcosecurity/falco-website/pull/184/files#diff-d7e08ba2003658dbc0eac816ae6939c4R114-R146)). 
We have to well explain to users how to use all images, IMHO.
Maybe we need a futher PR to improve that.

Reply:
See also https://github.com/falcosecurity/falco-website/issues/196